repairnator/repairnator-core/src/main/java/fr/inria/spirals/repairnator/BuildListener.java

```diff
@@ -0,0 +1,6 @@
+package fr.inria.spirals.repairnator;
+
+/**
+ * This class fetch build ids from ActiveMQ queue and run the pipeline with it.
```

Comment:
The interface is an abstract presentation, it should not be relevant to ActiveMQ. This line of comment refers to your `ScannerBuildListener`.